### ISSUE TITLE
EMACS tags table missing

### ISSUE THREAD

**Issue by eswenson1** [PDP-10/its#1723]
The file EMACS; EMACS TAGS should be present. It contains the tag table for emacs. It is not simply a matter of copying one from old tapes -- usually, the device will be incorrect (e.g. AI: or MC:).  We could find an old tags table, update the device to be DSK: and then run ":tags emacs;emacs<enter> to regenerate the tags table from the current sources.  

This will reveal one or more sources that are not present in DB/KA/KL, and which need to be retrieved or removed from the tags table.  We may wish to update the tags table to include all sources that we are including in emacs1; but that are not mentioned in our primordial tags file.

**Comment by larsbrinkhoff** [comment]
I'm wondering, what are the pros and cons of making a new TAGS file from scratch rather than updating an old one.

**Comment by larsbrinkhoff** [comment]
This is from the TAGS info.  I agree that starting from an old file is preferrable.

```
The normal mode of operation of the :TAGS program is to read in an
existing tag table and update it by rescanning the source files that it
describes.  The old tag table file itself tells :TAGS which source files
to process.  When making a new tag table you must start by making a
skeleton.  Then :TAGS is used to turn the skeleton into an accurate tag
table.

A skeleton tag table is like a real one except that it is empty; there
are no tags in it.  It contains exactly this much data, for each source
file that it is going to describe:

    <filenames>
    0,<language>
    ^_

**Comment by eswenson1** [comment]
Yes. What I did on my systems was start from the EMACS; EMACS TAGS in PI, change all the references to AI: to DSK:, and then run :TAGS EMACS;EMACS. This worked fine on ES. On EX/DB, it fails because of a missing source. Copying that source from ES to EX/DB allows it to succeed. 

I’ve successfully used the tags table since. I’ll create a PR soon.
